.github/CONTRIBUTING.md

```diff
@@ -133,7 +136,11 @@ logically independent patches which build on each other.  This makes large
 changes easier to review and approve which speeds up the merging process.
 * Try to keep pull requests simple. Simple code with comments is much easier
 to review and approve.
+* All proposed changes must be approved by a ZFS on Linux organization member.
+
+#### Tests and Benchmarks
 * Test cases should be provided when appropriate.
+"Appropriate" includes making sure new features have adequate code coverage
```

Comment:
How about, "This includes making sure new features have adequate code coverage."

Reply:
I like that phrasing.

Reply:
👍

Reply:
I'll leave it in for now, if more people have a preference for any of those, it can always be changed....

Reply:
@behlendorf You like the original or my proposed rewording?

Reply:
@rlaager shoot, I might have understoot @behlendorf  wrong... sorry >.<

I actually personally also prefer your rewording tbh :P